code-challenges.js

```diff
@@ -11,8 +11,18 @@ var people = [
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
 
-
-
+const sentCap = (arr) => {
+ let describe = arr.map( value => {
```

Comment:
you can 
```suggestion
 return arr.map( value => {
```
instead of doing a return on line 22 because this is the outer most loop